goaviatrix/vpn_user.go

```diff
@@ -36,10 +37,21 @@ type VPNUserInfo struct {
 	VpnUser VPNUser `json:"vpn_user"`
 }
 
+type GetVpnUsersResp struct {
+	Return  bool    `json:"return"`
+	Results VPNUser `json:"results"`
+	Reason  string  `json:"reason"`
+}
+
+type EditVPNUser struct {
```

Comment:
What is this structure for?

Reply:
It was for previous implementation. Deleted it.